README.md

```diff
@@ -32,6 +32,10 @@ it is required to also have [Buildx](https://docs.docker.com/build/install-build
 
 The Docker ACAP application is available as a **signed** eap-file in [Releases][latest-releases].
 
+> [!IMPORTANT]
+> On the Apps page it is required to toggle on `Allow root-privileged apps` to be able to install the Docker ACAP
+> In the Settings -> Account page, under SSH accounts it is also required to toggle off `Restrict root access` to be able to send the TLS certificates. Make sure to set the password of the root SSH user.
```

Comment:
Would be good to mention which firmware it affects. This is a suggestion

```suggestion
> [!IMPORTANT]
> From AXIS OS 11.8 `root` user is not allowed by default and in 12.0 it will removed. 
> Docker ACAP 1.X requires root and work is ongoing to create a version that does not.
> Meanwhile, the solution is to allow root to be able to install the Docker ACAP.
>
> On the web page of the device:
> 1. Go Apps page, toggle on `Allow root-privileged apps`.
> 1. Go to System -> Account page, under SSH accounts toggle off `Restrict root access` to be able to send the TLS certificates. Make sure to set the password of the `root` SSH user.
```